modules/stanford_courses_tag_translate/stanford_courses_tag_translate.module

```diff
@@ -0,0 +1,353 @@
+   */
+
+  $feeds_tamper = new stdClass();
+  $feeds_tamper->disabled = FALSE; /* Edit this to true to make a default feeds_tamper disabled initially */
+  $feeds_tamper->api_version = 2;
+  $feeds_tamper->id = 'stanford_course_importer-xpathparser_' . $xcount . '-clean_up_additional_xml';
+  $feeds_tamper->importer = 'stanford_course_importer';
+  $feeds_tamper->source = 'xpathparser:' . $xcount;
+  $feeds_tamper->plugin_id = 'find_replace_regex';
+  $feeds_tamper->settings = array(
+    'find' => '/<\\/?organization>|<\\/?tag>|<\\/?name>|\\s{2,}/',
+    'replace' => '',
+    'limit' => '',
+    'real_limit' => -1,
+  );
+  $feeds_tamper->weight = 1;
+  $feeds_tamper->description = 'Clean up additional XML';
+  $tampers['stanford_course_importer-xpathparser_' . $xcount . '-clean_up_additional_xml'] = $feeds_tamper;
+
+  $feeds_tamper = new stdClass();
+  $feeds_tamper->disabled = FALSE; /* Edit this to true to make a default feeds_tamper disabled initially */
+  $feeds_tamper->api_version = 2;
+  $feeds_tamper->id = 'stanford_course_importer-xpathparser_' . $xcount . '-replace_close_op_doub';
+  $feeds_tamper->importer = 'stanford_course_importer';
+  $feeds_tamper->source = 'xpathparser:' . $xcount;
+  $feeds_tamper->plugin_id = 'find_replace_regex';
+  $feeds_tamper->settings = array(
+    'find' => '/<\\/organization>\\s{1,}<name>/',
+    'replace' => '::',
+    'limit' => '',
+    'real_limit' => -1,
+  );
+  $feeds_tamper->weight = 0;
+  $feeds_tamper->description = 'Replace closing/opening tags with double-colon';
+  $tampers['stanford_course_importer-xpathparser_' . $xcount . '-replace_close_op_doub'] = $feeds_tamper;
+
+  $feeds_tamper = new stdClass();
+  $feeds_tamper->disabled = FALSE; /* Edit this to true to make a default feeds_tamper disabled initially */
+  $feeds_tamper->api_version = 2;
+  $feeds_tamper->id = 'stanford_course_importer-xpathparser_' . $xcount . '-course_translate_tamper';
+  $feeds_tamper->importer = 'stanford_course_importer';
+  $feeds_tamper->source = 'xpathparser:' . $xcount;
+  $feeds_tamper->plugin_id = 'stanford_courses_tag_translate_tamper';
+  $feeds_tamper->settings = array();
+  $feeds_tamper->weight = 2;
+  $feeds_tamper->description = 'Translate into human readable values';
+  $tampers['stanford_course_importer-xpathparser_' . $xcount . '-course_translate_tamper'] = $feeds_tamper;
+
+}
+
+
+/**
+ * Implements hook_form_alter().
+ */
+function stanford_courses_tag_translate_form_node_form_alter(&$form, &$form_state) {
+
+  /**
+   * Only node forms.
+   */
+  if (!isset($form["#node_edit_form"]) || !$form["#node_edit_form"]) {
+    return;
+  }
+
+  /**
+   * Only stanford_course node forms.
+   */
+  if (!isset($form["type"]["#value"]) || $form["type"]["#value"] !== "stanford_course") {
+    return;
+  }
+
+  // Disable the sucka so that people are not tempted to edit them.
+  $form["field_s_course_ec_tag"]["#disabled"] = TRUE;
+  $form["field_s_course_tag_human"]["#disabled"] = TRUE;
```

Comment:
Oh, I see what's happening here. Hm. This UX is a little janky. We should probably hide this field on the node display, and then on the edit form, add help text indicating why they can't edit it. But if we hide the field on node display, we should probably change the term reference field from autocomplete to a different widget, so that the user can see what human-readable tags are on the course.

This work can be a separate ticket. I'll make it.
![2015-12-15-11320-screenshot](https://cloud.githubusercontent.com/assets/821106/11820960/08dc03e4-a31c-11e5-92f0-1bd049ae084e.png)
![2015-12-15-11327-screenshot](https://cloud.githubusercontent.com/assets/821106/11820961/08de2e94-a31c-11e5-9393-574818794095.png)